doc/nrf/introduction.rst

```diff
@@ -3,16 +3,50 @@
 About the |NCS|
 ###############
 
-The |NCS| is hosted, distributed, and supported by Nordic Semiconductor for product development with Nordic SoCs.
-The SDK is distributed in multiple Git repositories with application code, drivers, re-distributable libraries, and forks of open-source code used in the SDK.
+The nRF Connect SDK (NCS) enables you to develop applications for nRF Series devices.
```

Comment:
replace 'for nRF Series devices' with 'for nRF52, nRF53, and nRF91 Series devices.'